includes/init.php

```diff
@@ -347,9 +363,17 @@ function print_trailer( $include_nav_links = true, $closeDb = true,
   }
 
   return $ret . '
-<!-- ' . $GLOBALS['PROGRAM_NAME'] . '     ' . $GLOBALS['PROGRAM_URL'] . ' -->
-' // Adds an easy link to validate the pages.
-  . ( $DEMO_MODE == 'Y' ? '
+<!-- ' . $GLOBALS['PROGRAM_NAME'] . '     ' . $GLOBALS['PROGRAM_URL'] . ' -->' .
+    ( $ALLOW_HTML_DESCRIPTION !== 'Y' ? '' :
+    /* Your choices here are "basic", "standard" or "full". */ '
+    <script src="//cdn.ckeditor.com/4.6.0/basic/ckeditor.js"></script>
```

Comment:
We need to ask somewhere;
1) Do they want to use it?
2) Call from CDN?
3) or, What local directory should we look for it in?

I prefer somewhere outside of WebCalendar where it's accessible from other programs.
I just hate having twenty copies of the same file scattered all over my site.